src/map/entities/charentity.cpp

```diff
@@ -1443,6 +1443,14 @@ bool CCharEntity::IsMobOwner(CBattleEntity* PBattleTarget)
         return true;
     }
 
+    for (auto& trust : PTrusts)
+    {
+        if (PBattleTarget->m_OwnerID.id == trust->id)
```

Comment:
Is there any mechanism setting the trust as the mob owner? AFAIK right now only a player can be the owner of a mob, and this ID is used in packets so it probably shouldn't be set to a trust. This branch could really use a merge of the claim code.

Reply:
should be the trusts "master" like with pets, even if they do appear on the party roster visually they aren't players and we need to be cautious what we send their way.

Reply:
So, not in this PR, but I wanted to sneak this in ahead of my _next_ PR. In the next PR Trusts are using full player abilities with all the trimmings. When Naji 'vokes the player would be locked off of the mob without this snippet.

Reply:
Anything other than players should never be set as the owner of a mob. Instead, it should be setting the player as the owner when the trust acts, avoiding this problem completely. This stuff is all centralized in the claim code that needs to get merged into this branch.